packages/testing/src/index.ts

```diff
@@ -81,10 +82,21 @@ export const setupContextWithConfig = async ({ timeout, ...config }: SetupConfig
 
   await api.isReady
 
+  const wsClient = new WebSocket(url)
+  const isWsClientReady = new Promise((resolve) => {
+    wsClient.on('open', () => {
+      resolve(true)
+    })
+  })
+  await isWsClientReady
+
   return {
     url,
     chain,
+    /** @polkadot/api WsProvider */
     ws,
+    /** WebSocket Client */
+    wsClient,
```

Comment:
do we need this?

Reply:
not sure if there are other ways to test this in e2e test